generic/tests/netperf.py

```diff
@@ -107,6 +106,37 @@ def env_setup(session, ip, user, port, password):
         remote.scp_to_remote(ip, shell_port, username, password,
                              agent_path, "/tmp")
 
+    def mtu_set(mtu):
+        """
+        Set server/client/host's mtu
+
+        :param mtu: mtu value to be set
+        """
+
+        linux_mtu_cmd = "ifconfig %s mtu %s"
+        error_context.context("Changing the MTU of guest", logging.info)
+        if params.get("os_type") == "linux":
+            ethname = utils_net.get_linux_ifname(server_ctl, mac)
+            ssh_cmd(server_ctl, linux_mtu_cmd % (ethname, mtu))
+        elif params.get("os_type") == "windows":
+            connection_id = utils_net.get_windows_nic_attribute(
+                            server_ctl, "macaddress", mac, "netconnectionid")
+            windows_mtu_cmd = 'netsh interface ipv4 set subinterface "%s" mtu=%s'
```

Comment:
Maybe you can use a `mtu_cmd ` here for both Linux and Windows by specifying  `mtu_cmd ` in variants for different system.

Reply:
yes, agree.

Reply:
we need also set mtu on clinet(127 line) and host (136 line), they are all linux system, so can not abstract to mtu_cmd.

Reply:
@hereischen Are linux_mtu_cmd/windows_mtu_cmd hardcode here? Put them in cfg?

Reply:
I think it‘s better put it into cfg

Reply:
It make sense, I will update it.